http-gateway/web/src/containers/things/things-i18n.js

```diff
@@ -152,7 +166,8 @@ export const messages = defineMessages({
   commandOnResourceExpired: {
     id: 'things.commandOnResourceExpired',
     defaultMessage: 'command on resource {deviceId}{href} has expired.',
-    description: 'Continuos message for command expiration, keep the first letter lowercase!',
+    description:
+      'Continuos message for command expiration, keep the first letter lowercase!',

```

Comment:
Continuous message for command expiration, keep the first letter lowercase!

Reply:
Not sure what this sentence means, and there is a typo.

Reply:
Its not a message, it's just a description of the translate message (for the translators).